src/settings.rs

```diff
@@ -9,6 +11,21 @@ pub struct Settings {
     pub database_pool_max_size: Option<u32>,
     #[cfg(test)]
     pub database_use_test_transactions: bool,
+    pub master_token_secret: Vec<u8>,
+}
+
+impl Default for Settings {
```

Comment:
`Default` is required for the implementation of `FromRequest` on `HawkPayload`.

Reply:
The FromRequest Config's a little odd, it always gets the from_request() cfg via Config::default() (so HawkPayloads from_request will always get this bad master_token_secret returned here). I guess it's more of a hardcoded (at build time) configuration

What we should do instead is put Settings (or some subset of it) in the actix_web state and from_request can get it from req.state()